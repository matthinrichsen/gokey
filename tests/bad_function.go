package tests

import (
	"errors"
	"log"

	"github.com/matthinrichsen/gokey/tests/anotherPackage"
)

func lookAtMyTerribleFunction() {
	a := anotherPackage.LastStruct{2}
	s := MyStruct{1, 2, 3, map[string]string{}, []int{}, MyStruct2{MyStruct3{errors.New(`implements Error()`)}}, anotherPackage.LastStruct{1}, a}
	log.Println(s, a)
}
