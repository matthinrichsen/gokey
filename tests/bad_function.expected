package tests

import (
	"errors"
	"log"

	"github.com/matthinrichsen/gokey/tests/anotherPackage"
)

func lookAtMyTerribleFunction() {
	a := anotherPackage.LastStruct{Int: 2}
	s := MyStruct{a: 1, b: 2, c: 3, m: map[string]string{}, arr: []int{}, MyStruct2: MyStruct2{MyStruct3{errors.New(`implements Error()`)}}, LastStruct: anotherPackage.LastStruct{1}, LS: a}
	log.Println(s, a)
}
